test/index.ts

```diff
@@ -18,11 +18,11 @@ export function run(): Promise<void> {
 			}
 
 			// Add files to the test suite
-			files.forEach(f => mocha.addFile(path.resolve(testsRoot, f)));
+			files.forEach((f) => mocha.addFile(path.resolve(testsRoot, f)));
 
 			try {
 				// Run the mocha test
-				mocha.run(failures => {
+				mocha.run((failures) => {
```

Comment:
i would love to change this lint rule, but we don't have to do it now - thoughts?

Reply:
There are 2 additional branches I have stacked from here. Once those are all merged in so the big moving around is done I'm all for changing it back

Reply:
sweet